nuitka/plugins/standard/standard.nuitka-package.config.yml

```diff
@@ -1184,11 +1181,10 @@
     patterns:
       - "bin/*.h"
   dlls:
-    - dest_path: "torch/lib"
-      dir: "lib"
-      include_from_filenames: ""
-      patterns:
-        - ""
+    - relative_path: "lib"
+      from_filenames:
+        "prefixes":
```

Comment:
The formatting doesn't seem to work properly anymore due to the restructuring.

Reply:
I think we have a bias there, that below a certain value it tends to be user provided values, rather than keywords. That might have to go, everything that is in a `oneOf` or something like that, should not be quoted.